### PR TITLE
feat: add `get oauth2-config` command

### PR DESCRIPTION
**Pull request by aeneasr**
## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md)
      and signed the CLA.
- [ ] I have referenced an issue containing the design document if my change
      introduces a new feature.
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [ ] I have added tests that prove my fix is effective or that my feature
      works.
- [ ] I have added necessary documentation within the code base (if
      appropriate).

### CODE REVIEW

File: cmd/cloudx/project/get_oauth2_config.go [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+	"github.com/ory/x/cmdx"
+)
+
+func NewGetOAuth2ConfigCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "oauth2-config <project-id>",
+		Aliases: []string{"oc", "oauth2-config"},
+		Args:    cobra.ExactArgs(1),
+		Short:   "Get an Ory Cloud project's OAuth2 configuration",
+		Long:    "You can use this command to render Ory OAuth2 Configuration as well.",
```
Comment (kevgo) [inline cmd/cloudx/project/get_oauth2_config.go:16]: The `Long` description as written here ("you can also ...") only makes sense if the reader read the `Short` description first. But Cobra doesn't do that. It shows the `Short` description in the overview of all commands and the `Long` description when showing help for a specific command. I therefore suggest to copy the short description into the long one:

```suggestion
		Short:   "Get an Ory Cloud project's OAuth2 configuration",
		Long:    "Get an Ory Cloud project's OAuth2 configuration. You can use this command to render Ory OAuth2 Configuration as well.",
```

Comment (aeneasr) [reply cmd/cloudx/project/get_oauth2_config.go:16]: Nice find, thank you!

Comment (aeneasr) [reply cmd/cloudx/project/get_oauth2_config.go:16]: I think we do this incorrectly for all commands. Would you be open to make a PR to fix all of the help commands? :)

Comment (kevgo) [reply cmd/cloudx/project/get_oauth2_config.go]: yes, created #199 to track this

**Review by kevgo — APPROVED** [review]

**Pull request merged**